Gemfile

```diff
@@ -1,2 +1,5 @@
+ruby '3.0.2'
 source 'https://rubygems.org'
+#will be updated in .gemspec once newer gem version is released
+gem 'conjur-cli', git: 'https://github.com/cyberark/conjur-cli.git', branch: 'bump-version-6.2.6'
```

Comment:
Should be changed after cli release

Reply:
We shouldn't be pinning to a branch like this, and the gemspec file is what is used for defining dependencies in debify.  The associated PRs for conjur-cli have been merged.  This should be removed and the conjur-cli released before merging.

Reply:
Released `conjur-cli`, updated the Gemfile